datarepo-clienttests/src/main/java/runner/config/ApplicationSpecification.java

```diff
@@ -5,6 +5,7 @@
   public int maxBulkFileLoad = 1000000;
   public int loadConcurrentFiles = 4;
```

Comment:
Should we bump this up to the new values i have in application properties? It would be 80

Reply:
Works for me!